### ISSUE TITLE
Slow work with Level strip

### ISSUE THREAD

**Issue by FerrLui** [tahoma2d/tahoma2d#705]
**When i`m turning on sevel strip from Panel menu program start to work very slowly, and almost everytime it crashes.**
If You need i can rec a video.
**To Reproduce**
List steps to reproduce the behavior.

1.Easily turn on Level strip panel with some drawings

**Device Information:**
 - OS: Windows 7
 - Tahoma2D Version:1.2

**Comment by artisteacher** [comment]
Are you saying you add multiple level strip panels? A video would be helpful.

**Comment by FerrLui** [comment]
@artisteacher Yes.When i want to open the level strip panel its okey, but when im working with pannel on  2-3  minutes its start to work very slowly and crash.((

**Comment by artisteacher** [comment]
@FerrLui - I haven't noticed the level strip slowing things down or crashing myself. We need more information in order to help. 
- Do you dock the level strip or leave it floating?
- Does this happen for every level type?
- Are you using vector guided drawing and/or the inbetweener?
-  Is this a problem for every scene and every project?
- Does the brush/style make any difference?
- Can you make a screen recording showing all the steps you follow?
- Can you share the crash report? (Should be found at: C:\Users\yourusername\AppData\Local\Tahoma2D\Tahoma2D\cache\crashrpt\)
